src/main/java/io/fabric8/jenkins/openshiftsync/BuildSyncRunListener.java

```diff
@@ -271,6 +280,10 @@ private void upsertBuild(Run run, RunExt wfRunExt) {
           nodeLinks.getLog().setHref(joinPaths(rootUrl, nodeLinks.getLog().href));
         }
       }
+      StatusExt status = stage.getStatus();
```

Comment:
c)  some other need deemed useful: Annotate the Build with pending input JSON so consoles can do the Proceed/Abort stuff if they want.